pkg/commands/git_commands/stash_loader.go

```diff
@@ -75,8 +75,23 @@ func (self *StashLoader) getUnfilteredStashEntries() []*models.StashEntry {
 }
 
 func (c *StashLoader) stashEntryFromLine(line string, index int) *models.StashEntry {
-	return &models.StashEntry{
+	model := &models.StashEntry{
 		Name:  line,
 		Index: index,
 	}
+
+	tstr, msg, ok := strings.Cut(line, "|")
+	if !ok {
+		return model
+	}
+
+	t, err := strconv.ParseInt(tstr, 10, 64)
+	if err != nil {
+		return model
+	}
+
+	model.Name = msg
```

Comment:
This line should move up to before parsing the int.

I actually prefer the version that I posted earlier (with the nested "if" statements instead of the early returns), it's fewer lines of code and many people find early returns objectionable in general. (I don't, necessarily, just in this case 😄.) I realize this is a matter of taste though.

Reply:
Sorry for the delay, @stefanhaller. I structured that function to return the original information as a raw message if any parsing error was raised (even if it's partial) - I didn't want to lose the not-parsable piece of it. I know it should not be possible to have errors, but I wanted to write it as safe as possible. Do you still prefer to move that line up?

I "unrolled" it to follow the go principle to left-align the happy path, instead of nesting those conditionals. It should make the code more readable and idiomatic. If it's fine for you, I'd keep it as is, otherwise, I'm happy to change it. Let me know!

Reply:
> return the original information as a raw message if any parsing error was raised (even if it's partial)

OK, fine with me. I guess  it comes down to which of these is more likely: `12345_678|This is the subject` (in which case you _do_ want to assign `msg`) vs. `Here's a subject that contains a | character` (in which case you don't). But since neither should ever happen in practice, I don't think it's too important.

> I "unrolled" it to follow the go principle to left-align the happy path, instead of nesting those conditionals.

I wasn't aware of that go principle, and that it's stronger than avoiding early returns. It's fine with me in this case.

Going to merge; thanks again!

Reply:
Thanks for merging and for all the feedback! 🙏